packages/api/schema/stryker-core.json

```
@@ -176,7 +176,7 @@
             "type": "string"
           },
           "default": {
-            "**/*+(.js|.ts|.cjs|.mjs)?(x)": "/* eslint-disable */\n// @ts-nocheck\n"
+            "**/!(.jest).{js|ts|jsx|tsx|cjs|mjs},!**/__test__/**": "/* eslint-disable */\n// @ts-nocheck\n"
```

This is definitely wrong. I think the following would work if comma-separated patterns were supported:
```suggestion
            "**/!(*.jest,*.spec,*.test).{js,ts,jsx,tsx,cjs,mjs},!**/__test__/**,!**/test/**,!**/spec/**": "/* eslint-disable */\n// @ts-nocheck\n"
```